homeassistant/components/xiaomi_miio/fan.py

```diff
@@ -345,6 +346,7 @@
     ATTR_DRY: "dry",
     ATTR_FAHRENHEIT: "fahrenheit",
     ATTR_MOTOR_SPEED: "motor_speed",
+    ATTR_WATER_LEVEL: "water_level",
```

Comment:
This should be a separate sensor entity.

See:
https://github.com/home-assistant/core/pull/39398#discussion_r492523276
https://github.com/home-assistant/core/pull/44252
https://github.com/home-assistant/core/pull/44372